data/decks/default.lua

```diff
@@ -0,0 +1,40 @@
+mesh = "../meshes/beam-hex.mesh"
```

Comment:
Do we want to logically connect this with the refinement levels in a `mesh` table?

Reply:
(I'm thinking these lua tables are kind of like input deck blocks)

Reply:
do you want something like:

```
mesh = {
  path = "../meshes/beam-hex.mesh",
  
  -- serial and parallel refinement levels
  ser_ref_levels = 0,
  par_ref_levels = 0,
}
```